addons/website_sale/static/src/js/variant_mixin.js

```diff
@@ -39,7 +39,18 @@ VariantMixin._onChangeCombination = function (ev, $parent, combination) {
         $product.data('product-tracking-info', combination['product_tracking_info']);
         $product.trigger('view_item_event', combination['product_tracking_info']);
     }
-
+    const addToCart = $parent.find('#o_wsale_cta_wrapper');
+    const contactUsButton = $parent.find('#contact_us_wrapper');
+    const productPrice = $parent.find('.product_price');
+    if (!combination.price) {
+        productPrice.removeClass('d-inline-block').addClass('d-none');
+        addToCart.removeClass('d-flex').addClass('d-none');
+        contactUsButton.removeClass('d-none').addClass('d-flex');
```

Comment:
I'm never sure about jQuery things, does this work whenever the option is disabled?
Will it now throw an error?

Reply:
I'm also not sure about jQuery things, I'm very new to them
To my understanding the option just controls the presence of contact us button.
The add to cart button and price will be hidden anyway.

Reply:
What I mean is if contactUsButton is not found, will removeClass not throw an error?